documentation/asciidoc/computers/config_txt/boot.adoc

```diff
@@ -29,7 +29,7 @@ If set to 1, the kernel will be started in 64-bit mode. Setting to 0 selects 32-
 
 In 64-bit mode, the firmware will choose an appropriate kernel (e.g. `kernel8.img`), unless there is an explicit `kernel` option defined, in which case that is used instead.
 
-Defaults to 1 on Pi 4s (Pi 4B, Pi 400, CM4 and CM4S), and 0 on all other platforms. However, if the name given in an explicit `kernel` option matches one of the known kernels then `arm_64bit` will be set accordingly.
+Defaults to 1 on Raspberry Pi 4, 5, 400 and Compute Module 4, 4S platforms. Defaults to 0 on all other platforms. However, if the name given in an explicit `kernel` option matches one of the known kernels then `arm_64bit` will be set accordingly.
```

Comment:
I'd add CM5 as well - everyone knows its coming.

Reply:
I try to avoid mentioning any potential products ahead of a formal announcement, especially if I've structured the text such that any new products could be trivially included in the line.

Reply:
Agree with Tom in principle. But for my own edification: I assumed that CM5 wouldn't support 32-bit mode at all, just like Pi 5. So I don't think we need to add it here at release. Is that correct, @JamesH65 ? Or will CM5 support 32-bit mode?

Reply:
I believe it's actually the SoC (and not the specific Model) that determines if it's capable of booting 32-bit kernel only (BCM2835, BCM2836), either 32- or 64-bit kernel (BCM2837, RP3A0, BCM2711), or 64-bit kernel only (BCM2712) ?

But (in general) we explicitly list the Models in documentation rather than SoCs, because people find it easier to identify their Model than to identify their SoC :socks: :wink: